### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by Uchenna-ibute**
Adding the styles of your choice to the page.

### CODE REVIEW

**Comment by DeliceLydia** [comment]
## Hi @Uchenna-ibute, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional README Please replace field from the template  with your personal information instead of submitting the template only

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -93,54 +95,38 @@ Clone this repository to your desired folder:
...
 ```
---->
 
 ### Deployment
 
-You can deploy this project using:
-
-<!--
-Example:
+You can deploy this project using:[Github pages](https://docs.github.com/en/pages) or any hosting service of your choice.
```
Comment (muneebulrehman) [inline README.md:128]: - [ ] Kindly mention 1-3 future features to the `Future features` section.
- [ ] Kindly add the API that your app consumes to the README file. ⬇️
image

File: src/style.css [hunk 2]
```diff
@@ -29,3 +32,18 @@ h1 {
...
+
+  .container {
+    font-size: 8px;
+    margin: 20px;
+  }
+
+  input {
+    width: 70%;
+  }
+}
```
Comment (muneebulrehman) [inline src/style.css:93]: The styling of your application could be improved a lot. Right now, the layout is bland and barely matches the wireframe provided by the Microverse. Kindly work on the styling of your application. It doesn't have to be flashy, but still you can improve the layout and UI. Eg., the scoreboard has a border around it in the wireframe, you can consider adding that. Give some margin between the input taking part. Move the top heading to the left side.
| Your design  | Wireframe design |
| ------------- | ------------- |
| image| image|

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Uchenna-ibute , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- All technical requirements met.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Uchenna-ibute) [reply README.md:128]: Hello thank You for the feedback, but please can you explain more about what u mean in this comment?

**Review by Deepakdanger — APPROVED** [review]
Hi @Uchenna-ibute , 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**